src/workerd/api/pyodide/pyodide.c++

```diff
@@ -105,22 +93,39 @@ kj::Array<kj::String> _getWorkerFiles(
   return builder.releaseAsArray();
 }
 
-kj::HashSet<kj::String> _getNames(kj::ArrayPtr<kj::String> names) {
+kj::HashSet<kj::String> PythonModuleInfo::getWorkerModuleSet() {
   auto result = kj::HashSet<kj::String>();
-  for (auto& name: names) {
-    if (!name.endsWith(".py")) {
+  const auto vendor = "vendor/"_kj;
+  const auto dotpy = ".py"_kj;
+  const auto dotso = ".so"_kj;
+  for (auto& name_: names) {
```

Comment:
Nit: why the underscore suffix?

Reply:
I need to turn it into a string pointer, so I immediately do:
```
kj::StringPtr name = name_;
```
on the next line.